playbook/app/pb_kits/playbook/pb_legend/docs/example.yml

```diff
@@ -1,11 +1,13 @@
 examples:
-  
+
   rails:
   - legend_default: Default
   - legend_prefix: With Prefix Text
- 
-  
-  
+  - legend_status_colors: With Status Colors
```

Comment:
I would love to see more documentation around the allowed colors that can be passed to this component, since it seems a little unintuitive about what they accept

Reply:
We can do that.